frontend/src/app/[locale]/formPrototype/[applicationId]/page.tsx

```diff
@@ -1,4 +1,5 @@
 import { Metadata } from "next";
```

Comment:
I know we probably don't have design input on this yet which is why it hasn't been done, but a path setup like
`formPrototype/application/:applicationId/form/:formId would make a lot more sense, and we can probably lead the charge on that even without finished designs. As long as that is actually what this represents?

Reply:
I'll add changing the route to the cleanup ticket. A bigger concern after the demo is using UUIDs for route params, as it quickly gets ugly for the user once we are a couple of params deep.

Reply:
yeah, agreed - seeing those two uuids mashed up against each other in the url was what made me think of this